addons/60fps/addon.json

```diff
@@ -1,10 +1,15 @@
 {
-  "name": "60FPS project player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom project framerate",
+  "description": "Alt+Click the green flag to toggle between Scratch's default 30FPS and a custom framerate.",
   "info": [
+    {
+      "type": "info",
+      "text": "The framerate (FPS) determines how many blocks are run per second. For example, running at 60FPS doubles the execution speed.",
+      "id": "fpsExplanation"
+    },
     {
       "type": "notice",
```

Comment:
I'd even understand if the `projectRunsFasterNotice` note was marked as a red warning. That might catch attention a bit _too_ much depending on who you ask, though.